code/modules/clothing/clothing.dm

```diff
@@ -480,6 +480,12 @@
 /obj/item/clothing/shoes/proc/on_kick(mob/living/user, mob/living/victim)
 	return
 
+/obj/item/clothing/shoes/kick_act(mob/living/carbon/human/user)
+	if(prob(15) && (user.equip_to_slot_if_possible(src, slot_shoes) == 1))
```

Comment:
you don't need the == 1, since it can only return 0 or 1, and 0 is not true

also I would just make this 100%, it's an easter egg, how often are you kicking shoes without them on?

Reply:
> you don't need the == 1, since it can only return 0 or 1, and 0 is not true

I haven't written in DM in a while and forgot if it was advanced enough to not require the "==1". Thanks. 
> also I would remove the chance, it's an easter egg, how often are you kicking shoes without them on?

Yeah, why not. Just followed the feature-request.